modules/accredited_representative_portal/app/services/filter_service.rb

```diff
@@ -0,0 +1,48 @@
+class FilterService
+  def self.handle_filter(filter_params)
+    filter_params[:status] ||= 'Pending'
+    filter_params[:sortField] ||= 'ar_power_of_attorney_requests.created_at'
+    filter_params[:sortDirection] ||= 'desc'
+    filter_params[:pageNumber] ||= 0
+    filter_params[:pageSize] ||= 10
```

Comment:
Since dry-schema doesn't allow default values, let's place all the defaults in a frozen constant like so:
```
  DEFAULT_PARAMS = {
    status: 'Pending',
    sortField: 'ar_power_of_attorney_requests.created_at',
    sortDirection: 'desc',
    pageNumber: 0,
    pageSize: 10
  }.freeze
```
Then go ahead and turn the service into an object where you can merge the defaults in during initialization where you do the normalization.